.vs/launch.vs.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
This file contains some program arguments configurations, that's why I decided to commit it. It's removed now.